modules/pytest/Makefile

```diff
@@ -1,14 +1,50 @@
-pytest: pytest/test
+PYTEST = $(PIPENV_RUN) pytest
+
+.PHONY: pytest/init
+pytest/init: pipenv reports/
+
 .PHONY: pytest
+## Run Python tests with pytest
+pytest: pytest/init ; $(PYTEST)
```

Comment:
Can you put the commands on a separate line?

```Makefile
pytest: pytest/init
    $(PYTEST)
```
I think it's more readable that way; you can tell at a glance if something is a dependency target or a command.

Reply:
Personally, I like shortening, but I'm not that attached. Change made.